backend/controller/webAuthn.go

```diff
@@ -353,6 +354,15 @@ func GetYubiKeyDoorAccess(w http.ResponseWriter, r *http.Request) {
 	helpers.JsonWriter(w, "Login Success, opening door")
 }
 
+func DeleteSessionData(w http.ResponseWriter, r *http.Request) {
+	res := db.DB.Session(&gorm.Session{AllowGlobalUpdate: true}).Delete(&models.SessionData{})
```

Comment:
Why are we filtering by `AllowGlobalUpdate: true`?

Reply:
![image](https://github.com/lynx-locks/lynx-web/assets/57304476/652cc7c0-e404-4e4a-9d6f-f6ba962a172d)
sql is as follows
we are not filtering 
![image](https://github.com/lynx-locks/lynx-web/assets/57304476/358f727b-8ac2-445f-8392-2cd514169358)

Reply:
ah, gotcha. Thanks.

Reply:
its just basically creating a session with delete all perms since they are not normally enables could also be done with 

db.DB.exec("Delete FROM session_data")